plugins/@grouparoo/hubspot/src/lib/export-objects/exportRecords.ts

```diff
@@ -221,11 +219,15 @@ export const exportRecords: ExportRecordsPluginMethod = async ({
     });
   } catch (error) {
     if (error?.response?.status === 429) {
-      let retryIn = error?.response?.headers["Retry-After"]; // seconds
```

Comment:
Is this not being returned in the response?

Reply:
yeah, that must have been from some other plugin.
There are a bunch of headers but none of them _quite_ say when to retry 🤷